Misha/Files/mp3_tagger/tagger_main.py

```diff
@@ -0,0 +1,45 @@
+import eyed3
+
+
+def input_mp3_file_name(mp3_file_name):
+    file_split = str(mp3_file_name).split('.')
+    if len(file_split) == 1 or file_split[len(file_split)-1] != 'mp3':
+        raise TypeError('Invalid file type')
+    else:
+        try:
+            audiofile = eyed3.load(mp3_file_name)
+            return audiofile
+        except OSError:
+            raise OSError('file not found')
+
+
+def get_tag(audiofile):
+    tags_dict = {'artist': audiofile.tag.artist,
+                 'album': audiofile.tag.album,
+                 'album artist': audiofile.tag.album_artist,
+                 'title': audiofile.tag.title,
+                 'track num': audiofile.tag.track_num}
+    return tags_dict
+
+
+def set_tag(audiofile, tags_dict):
+    try:
+        audiofile.tag.artist = tags_dict['artist']
+        audiofile.tag.album = tags_dict['album']
+        audiofile.tag.album_artist = tags_dict['album artist']
+        audiofile.tag.title = tags_dict['title']
+        audiofile.tag.track_num = tags_dict['track num']
+        audiofile.tag.save()
+    except KeyError:
+        raise KeyError
+
+
+def main():
```

Comment:
Please add tests for main function

Reply:
i know tha i should use mock and patch for this. I dont understand how, but i know where. It stops me